Firmata.h

```diff
@@ -15,7 +15,7 @@
 #define Firmata_h
 
 #include "Boards.h"  /* Hardware Abstraction Layer + Wiring/Arduino */
-#include "FirmataConstants.h"
+#include "FirmataDefines.h"
```

Comment:
This split allows big project users to have all the constants under the firmata namespace, while Arduino users get to keep the global preprocessor defines.

The global preprocessor defines are based off the constants, so there is a single point of reference and the defines can be removed at a later date (v3.0) if desired.